cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,36 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomePage from '../support/pages/HP.pageObject';
+import LaptopsPage from '../support/pages/laptops.pageObject';
+import CartPage from '../support/pages/cart.pageObject';
+import OrderPage from '../support/pages/makeOrder.pageObject';
 
-  });
+describe('Purchase Laptop', () => {
+  const homePage = new HomePage();
+  const laptopsPage = new LaptopsPage();
+  const cartPage = new CartPage();
+  const orderPage = new OrderPage();
+
+  it('Should make the purchase', () => {
+    homePage.visit();
+    homePage.clickLaptops();
+    laptopsPage.clickTheCategory();
```

Comment:
```suggestion
    laptopsPage.clickTheCategory('category name');
```
make your method reusable